sympy/stats/crv_types.py

```diff
@@ -3550,6 +3552,65 @@ def Reciprocal(name, a, b):
     """
     return rv(name, ReciprocalDistribution, (a, b))
 
+#-------------------------------------------------------------------------------
+# Rician distribution --------------------------------------------------------
+
+class RicianDistribution(SingleContinuousDistribution):
+    _argnames = ('alpha', 'beta')
+
+    @property
+    def set(self):
+        return Interval(0, oo)
+
+    @staticmethod
+    def check(alpha, beta):
+        _value_check(alpha > 0, "Shape parameter alpha must be positive.")
+        _value_check(beta > 0, "Shape parameter beta must be positive.")
+
+    def cdf(self, x):
+        alpha, beta = self.alpha, self.beta
+        return marcumq(1, alpha/beta, x/beta)
+
+    def pdf(self, x):
+        alpha, beta = self.alpha, self.beta
+        return exp(-(x**2 + alpha**2)/(2*beta**2)) * x * besseli(0, (x*alpha)/(beta**2))
```

Comment:
The pdf seems to be incorrect. I think it should be divided further by `beta**2`. See:[https://en.wikipedia.org/wiki/Rice_distribution#Characterization]( https://en.wikipedia.org/wiki/Rice_distribution#Characterization)

Reply:
You're correct I missed that one. I'll add this shortly.

Should I also add a numerical example to make sure it's working correctly?